src/Core/Geographic/Extent.js

```diff
@@ -424,12 +424,16 @@ class Extent {
      * @returns {Boolean}
      */
     intersectsExtent(extent) {
+        return Extent.intersectsExtent(this, extent);
```

Comment:
Why do you move it to static ? 
+ Could you add a deprecation warning on this method please?

Reply:
There is no need for a deprecation warning as it doesn't make the previous one deprecated.

Reply:
oh yes ok we can keep both :)